src/plugins/friendInvites/index.ts

```diff
@@ -25,16 +25,15 @@ const FriendInvites = findByPropsLazy("createFriendInvite");
 
 export default definePlugin({
     name: "FriendInvites",
-    description: "Create and manage friend invite links via slash commands (/create friend invite, /view friend invites, /revoke friend invites).",
-    authors: [Devs.afn, Devs.Dziurwa],
+    description: "Create and manage friend invite links via slash commands (/fvite create, /fvite list, /fvite revoke).",
```

Comment:
this wasnt caught by anyone else so imma point it out
these werent updated after you changed it from fvite to friendinvite
```suggestion
    description: "Create and manage friend invite links via slash commands (/friendinvite create, /friendinvite list, /friendinvite revoke).",
```

Reply:
oh mb